config/locales/fr/budgets.yml

```diff
@@ -4,9 +4,12 @@ fr:
       show:
         title: Votre vote
         amount_spent: Montant dépensé
-        remaining: "Il vous reste <span>%{amount}</span> à investir."
+        remaining:
```

Comment:
We only translate in English and Spanish in Github. 👌 The other translations we handle them through Crowdin, makes things simpler. Feel free to remove translations for other languages 👍

Reply:
Will remove...